### DISCUSSION TITLE
Android Emulator not working (shows default screen or previous app)

### DISCUSSION THREAD

**Discussion by joer33304** [unoplatform/uno#9096] [Q&A] [answered]
I am still in the super simple test phase and still only have problems :( I created a new solution from the template and added the code from the UWP data grid example. XAML starts and shows the grid. Android starts the emulator and shows just the home screen.

I created a second solution with no code changes and droid shows hello world.

I went back to the non working solution and now when I start up the emulator it also shows hello world and not the grid !

I cant find any HOWTO for the mobile stuff. I am especially also interested in what needs to be done to get an app to run on iPhone. Are there any resources and why is my droid app not showing ?

**Comment by jeromelaban** [comment]
Uno on Android is based on the Microsoft's own Xamarin.Android tooling, you may find information about [how to debug android applications here](https://docs.microsoft.com/en-us/xamarin/android/deploy-test/debugging/debug-on-emulator?tabs=windows).

**Comment by pkar70** [comment]
You can also try BlueStacks - seems like it works better than emulator from Xamarin.

**Comment by joer33304** [comment, marked as answer]
For anybody with the same problem, the issue was that deploy was not set for the mobile project. Make sure under Build/ConfigurationManager deploy is set for the mobile project

**Reply by jeromelaban** [reply]
Thanks for the troubleshooting! Did you create your project using `dotnet new` or using the Visual Studio "Project/new" UI ?

**Reply by joer33304** [reply]
Used visual studio new Uno Solution. However to debug issues I first manually removed all projects but wasm and gradually added projects to see if it still works. I guess that removed the deploy checkbox.

**Reply by jeromelaban** [reply]
Thanks, we'll track this in https://github.com/unoplatform/uno/issues/9120.
